politeiawww/api/v1/v1.go

```diff
@@ -194,6 +198,7 @@ var (
 		ErrorStatusMalformedUsername:           "malformed username",
 		ErrorStatusDuplicateUsername:           "duplicate username",
 		ErrorStatusVerificationTokenUnexpired:  "verification token not yet expired",
+		ErrorInvalidCensorMessage:              "Invalid censor message",
```

Comment:
Let's use `ErrorMalformedCensorMessage` and `"malformed censor message"` instead to stay consistent with the others.